### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by hashier**

### CODE REVIEW

File: website/learn/effective-elvish.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ lorem
 456
 ```
 
-Note that it is more concise to write `mygrep ... < in.txt`.
+**Note:** that it is more concise to write `mygrep ... < in.txt`.
```
Comment (krader1961) [inline website/learn/effective-elvish.md:226]: It seems like you have misunderstood what a notable "note" comment is and how it should be used. Also, even if appropriate here you should have done a s/that it is/It is/ substitution.

Comment (hashier) [reply website/learn/effective-elvish.md]: Fixed in 41ddf5ace14faa33521e500bded307fffe26fa7c

**Comment by krader1961** [comment]
@hashier, I haven't reviewed your changes since my previous feedback a day ago but this is, it seems to me, a case where you should squash all your commits to a single commit and force push the result upstream to Github. Or perhaps close the current P.R. and start a new one after squashing your closely related commits.

**Comment by xiaq** [comment]
> @hashier, I haven't reviewed your changes since my previous feedback a day ago but this is, it seems to me, a case where you should squash all your commits to a single commit and force push the result upstream to Github. Or perhaps close the current P.R. and start a new one after squashing your closely related commits.

You're entitled to your opinion, but I don't have one on this matter. GitHub makes it easy enough to squash the commits when merging the PR.

**Pull request merged**